mobile/src/components/billing/billing.tsx

```diff
@@ -1,12 +1,57 @@
+import { StripeProvider } from '@stripe/stripe-react-native';
 import React, { FC } from 'react';
 
-import { Text, View } from '~/components/common/common';
+import { DataStatus } from '~/common/enums/enums';
+import { BillingReplenishToken } from '~/common/types/types';
+import { Spinner } from '~/components/common/common';
+import { useAppDispatch, useAppSelector, useEffect } from '~/hooks/hooks';
+import { billingActions } from '~/store/actions';
+
+import {
+  DEFAULT_REPLENISH_AMOUNTS,
+  REPLENISH_PUBLIC_KEY,
+} from './common/common';
+import { PaymentScreen } from './components/components';
 
 const Billing: FC = () => {
+  const { billingDataStatus, userMoneyBalance } = useAppSelector(
+    ({ billing }) => ({
+      billingDataStatus: billing.dataStatus,
+      userMoneyBalance: billing.userMoneyBalance,
+    }),
+  );
+
+  const dispatch = useAppDispatch();
+
+  const handleWithdraw = (): void => {
+    dispatch(billingActions.withdraw({ userCurrentBalance: userMoneyBalance }));
+  };
+
+  const handleReplenish = (
+    amountOfMoneyToReplenish: number,
+    token: BillingReplenishToken,
+  ): void => {
+    dispatch(billingActions.replenish({ amountOfMoneyToReplenish, token }));
+  };
+
+  useEffect(() => {
+    dispatch(billingActions.getUserWithMoneyBalance());
+  }, [dispatch]);
```

Comment:
No need to put it in useEffect
```suggestion
  }, []);
```